hazelcast/proxy/map.py

```diff
@@ -237,6 +532,30 @@ def put_all(self, map):
         return combine_futures(*futures)
 
     def put_if_absent(self, key, value, ttl=-1):
+        """
+        If the specified key is not already associated with a value, associate it with the given value. If ttl is
```

Comment:
Associates the specified key with the given value if it is not already associated. If ttl is....